searchwidget/src/main/java/com/harsh/searchwidget/SearchBar.java

```diff
@@ -217,7 +217,7 @@ private void init(AttributeSet attributeSet) {
 
         destiny = getResources().getDisplayMetrics().density;
         if (adapter == null) {
-            adapter = new DefaultLocalSuggestionsAdapter(LayoutInflater.from(getContext()));
+            adapter = (SuggestionsAdapter<S, T>) new DefaultLocalSuggestionsAdapter(LayoutInflater.from(getContext()));
```

Comment:
This will not generate errors as default Local suggestions extends SuggestionsAdapter<S,T>